homeassistant/components/dsmr/sensor.py

```diff
@@ -32,47 +32,51 @@
 ICON_POWER_FAILURE = "mdi:flash-off"
 ICON_SWELL_SAG = "mdi:pulse"
 
-# Smart meter sends telegram every 10 seconds
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
-
 RECONNECT_INTERVAL = 5
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.string,
         vol.Optional(CONF_HOST): cv.string,
         vol.Optional(CONF_DSMR_VERSION, default=DEFAULT_DSMR_VERSION): vol.All(
-            cv.string, vol.In(["5", "4", "2.2"])
+            cv.string, vol.In(["5B", "5", "4", "2.2"])
         ),
         vol.Optional(CONF_RECONNECT_INTERVAL, default=30): int,
         vol.Optional(CONF_PRECISION, default=DEFAULT_PRECISION): vol.Coerce(int),
     }
 )
 
+# Smart meter sends telegram every 10 seconds or every second for DSMR 5
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=1)
```

Comment:
What's the benefit of polling every second? In a local setup it's not an issue but I doubt that the meters are that fast to detect if a light is on for 500 ms or so, also it only makes sense the current electrical usage but not for the total.

Reply:
Well the DSMR meter pushes a new data blob every second, with updated values.
Is it really needed ? Well no, but why wouldn't we allow it if we have the possibility ? :)

You prefer a configurable value with default 10seconds for example?

Reply:
And even something more important, the MIN_TIME_BETWEEN_UPDATES is used nowhere in the rest of the code. Removing it, as its just useless now.